win32ss/user/user32/controls/button.c

```diff
@@ -1066,9 +1041,15 @@ static void CB_Paint( HWND hwnd, HDC hDC, UINT action )
 
     GetClientRect(hwnd, &client);
     rbox = rtext = client;
-
-    checkBoxWidth  = 12 * GetDeviceCaps( hDC, LOGPIXELSX ) / 96 + 1;
-    checkBoxHeight = 12 * GetDeviceCaps( hDC, LOGPIXELSY ) / 96 + 1;
+    
+#ifdef __REACTOS__
+    //TODO: Let's actually fix this stupid thing at some point
+    checkBoxWidth  = 12 * 96 / 96 + 1;
```

Comment:
Using GetDeviceCaps here wouldn't work, (as in the original code)

Reply:
Probably but that’s not why I did this,

Im going to solve the real problem here this is just a note for me